src/components/org-users/views.tsx

```diff
@@ -291,72 +282,76 @@ export function PermissionTable(
-                  state={
-                    props.values.space_roles[space.metadata.guid].developers
-                  }
-                />
-              </td>
-              <td className="govuk-table__cell ">
-                <Permission
-                  name="Space auditor"
-                  checked={
-                    !!props.values.space_roles[space.metadata.guid].auditors
-                      .desired
-                  }
-                  namespace={`space_roles[${space.metadata.guid}][auditors]`}
-                  state={props.values.space_roles[space.metadata.guid].auditors}
-                />
-              </td>
-            </tr>
-          ))}
-        </tbody>
-      </table>
+            </dt>
+            <dd className="govuk-summary-list__value">
+              {space.entity.name}
+            </dd>
+          </div>
+          <div className="govuk-summary-list__row">
+            <dt className="govuk-summary-list__key">
+              Roles
+            </dt>
+            <dd className="govuk-summary-list__value">
+              <fieldset className="govuk-fieldset" aria-describedby="roles-hint">
+                <legend className="govuk-fieldset__legend">
+                <span className="govuk-visually-hidden">Which space roles would you like to assign to this user?</span>
+                </legend>
+                <span id="roles-hint" className="govuk-hint">
+                  Select all that apply.
+                </span>
+                <div className="govuk-checkboxes">
+                  <Permission
+                    name="Space manager"
+                    checked={
+                      !!props.values.space_roles[space.metadata.guid].managers
+                        .desired
+                    }
+                    namespace={`space_roles[${space.metadata.guid}][managers]`}
+                    state={props.values.space_roles[space.metadata.guid].managers}
+                  />
+                  <Permission
+                    name="Space developer"
+                    checked={
+                      !!props.values.space_roles[space.metadata.guid].developers
+                        .desired
+                    }
+                    namespace={`space_roles[${space.metadata.guid}][developers]`}
+                    state={
+                      props.values.space_roles[space.metadata.guid].developers
+                    }
+                  />
+                  <Permission
+                    name="Space auditor"
+                    checked={
+                      !!props.values.space_roles[space.metadata.guid].auditors
+                        .desired
+                    }
+                    namespace={`space_roles[${space.metadata.guid}][auditors]`}
+                    state={props.values.space_roles[space.metadata.guid].auditors}
+                  />
+                </div>
+              </fieldset>
+            </dd>
+          </div>
+        </dl>
+        </>
```

Comment:
```suggestion
        </Fragment>
```

Supplement to 287.

You may also need to import the `Fragment` from react.